src/semeio/forward_models/__init__.py

```diff
@@ -54,7 +58,15 @@ def validate_pre_realization_run(
         return fm_step_json
 
     def validate_pre_experiment(self, fm_step_json: ForwardModelStepJSON) -> None:
-        pass
+        errors = DesignKWScript.validate_configuration(fm_step_json.get("argList")[0])
+        should_fail_on_validation_error = (
+            fm_step_json.get("argList")[2].lower() == "true"
```

Comment:
For now, validate=False should hide the warning message, and the error never fail. The default value should be to warn.